TreeTracker/UI/Views/Profile Screen/ProfileViewModel.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ProfileViewModel.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 3/23/21.
+//  Copyright © 2021 Greenstand. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol ProfileViewModelCoordinatorDelegate: class {
+    func profileViewModel(_ profileViewModel: ProfileViewModel, changeUser planter: Planter)
+}
+
+protocol ProfileViewModelViewDelegate: class {
+    func profileViewModel(_ profileViewModel: ProfileViewModel, didFetchDetails details: ProfileViewModel.ProfileDetails)
+    func profileViewModel(_ profileViewModel: ProfileViewModel, didUpdateTrees trees: [Tree])
+    func profileViewModel(_ profileViewModel: ProfileViewModel, didReceiveError error: Error)
+}
+
+class ProfileViewModel {
+    weak var viewDelegate: ProfileViewModelViewDelegate?
+    weak var coordinatorDelegate: ProfileViewModelCoordinatorDelegate?
+    private let planter: Planter
+    private let treeMonitoringService: TreeMonitoringService
```

Comment:
Interested in why you have chosen to use `TreeMonitoringService` in this ViewModel? It doesn't look like you are starting it anywhere so it wont do anything. Maybe consider removing along with the delegate methods if we don't need it.

Reply:
I removed TreeMonitoringService